docs/current_docs/manuals/developer/python/constructor.mdx

```diff
@@ -102,9 +75,7 @@ a factory class method which must be named `create`:
 ```python file=./snippets/constructor/async/main.py
 ```
 
-[@object_type]: https://dagger-io.readthedocs.io/en/latest/module.html#dagger.object_type
-[dataclass]: https://docs.python.org/3/library/dataclasses.html
```

Comment:
There's a link requiring this, in "Exclude argument in constructor".

Reply:
Thanks for the catch, fixed